lib/solr_wrapper/settings.rb

```diff
@@ -20,6 +20,25 @@ def initialize(static_config)
       @static_config = static_config
     end
 
+    def as_json
```

Comment:
```suggestion
    def to_h
```

I don't think json-ness is necessary, and I could see this being reused to dump out an example yml config or something.